### ISSUE TITLE
An ad just played after I clicked on a video

### ISSUE THREAD

**Issue by calvincorrieri** [arichornlover/YouTubeRebornPlus#125]
### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Is there an existing issue/question for this?

- [X] I have searched the existing issues

### Do you think this is a bug?

❌ No, I don't think this is a bug. I will explain below

### My question

Not really a question, just bringing this to your attention. I believe YouTube may have changed the way ads are served. I was just watching YouTube using this ipa like I always do and suddenly I got a “video will play after ad” ad. I closed the video and opened it again and no ad played that time, but this was just a few minutes ago so idk if it’s a fluke or not.

### Additional context

It’s also possible that the ad playing was just a bug, but I’ve been using tweaked YouTube ipa’s for what feels like a decade at this point, and that’s never happened a single time before. I hope this doesn’t come off as criticism. I am incredibly grateful for the work that went into this ipa. Without it, I straight up wouldn’t watch YouTube at all. I just wanted to bring this to your attention in case YouTube made some changes server-side that break the adblock method.

**Comment by no7here** [comment]
I have also had this issue a few times. Reinstalled it thinking I'd butchered something, but I definitely have the ad blocker on.

**Comment by no7here** [comment]
> > I have also had this issue a few times. Reinstalled it thinking I'd butchered something, but I definitely have the ad blocker on.
> > Edit: I've got a clip of a pre-roll (idk if this is the right terminology, but an unskippable ad before the video) occurring, if it would be helpful, I can attach it.
> 
> I think it only does **pre-roll** ads because I have the same exact issue aswell, at the moment not sure if that can be solved since I don’t know where its triggering those ads. however I don’t think you’ll get **mid-roll** and **post-roll** ads but hopefully AdBlock would be improved in the future.

Yeah, I've only had pre-roll ads, not mid-roll or post-roll. Thankfully it's very rare though, and clicking off the vid and back on does get rid of them.
